applications/ila-cookie-banner/RELEASE.md

```diff
@@ -0,0 +1,94 @@
+# Cookie Banner Release Process
+
+This is a guide for installing the a new release of the campus cookie banner to the content delivery network (CDN) server S3 bucket.
+
+The target audience for this document is the team that maintains the CDN that serves the shared cookie banner.
+
+## Stage the New Release
+
+### Step 1. Create the new release folder.
+
+	Create a new folder, named for the versioned release. For example, if `1.0.0`:
+
+		https://onetrust.techservices.illinois.edu/1.0.0
+		
+### Step 2. Get the latest cookie banner zip file from GitHub Actions.
+
++ Visit https://github.com/app-illinois/Design-Resources/actions
++ Click on `Cookie banner zip`
```

Comment:
This can be a link:

```suggestion
Click on [`Cookie banner zip`](https://github.com/app-illinois/Design-Resources/actions/workflows/cookie-banner-zip.yaml)
```